src/Page/HomePage/index.js

```diff
@@ -14,8 +13,9 @@ import Digitallnitiatives from "./DigitalInitiatives";
 
 function Home() {
   const history = useHistory();
+  const screenWidth = window.innerWidth;
   return (
-    <div className="navgurukul-homepage ">
+    <div className="navgurukul-homepage container-xl ">
```

Comment:
@Apekshahande You gave container-xl still it doesn't appear how we want it to look like. Can you look for an alternative?

Reply:
Can you see the same dive in different sections, this says the size of that section. There must be somewhere the size of those sections has defined. You need to find it and increase the size of it.

![image](https://github.com/navgurukul/nightingale/assets/67674096/b4c64c99-915b-42ac-9004-e5c73a1c8a2e)